data/discovery/field/school-wls-school-type.yaml

```diff
@@ -0,0 +1,6 @@
+cardinality: '1'
+datatype: string
+field: school-wls-school-type
+phase: discovery
+register: school-wls
```

Comment:
I think `register: school-wls` should be `register: school-wls-school-type`.

Reply:
agreed